python3/electro_grammar.py

```diff
@@ -0,0 +1,37 @@
+from antlr4 import *
+from antlr.ElectroGrammarLexer import ElectroGrammarLexer
+from antlr.ElectroGrammarListener import ElectroGrammarListener
+from antlr.ElectroGrammarParser import ElectroGrammarParser
+
+
+class ElectroGrammarToObjectListener(ElectroGrammarListener):
+    def __init__(self):
+        self.obj = {}
+
+    def enterCapacitance(self, ctx):
+        cprefix_lookup = {'u': 10e-6, 'n': 10e-9, 'p': 10e-12}
+        number = float(str(ctx.NUMBER()))
+        cprefix = str(ctx.CPREFIX())
```

Comment:
I think you forgot to use the `cprefix_lookup`. I kind of want to make the conversion and normalisation part of the grammar but that may be awkward or impossible. 

Also, maybe we should be using [`decimal`](https://docs.python.org/3.6/library/decimal.html) for any values as I observed floating point errors in my previous python port.